code/modules/power/cables/cable.dm

```diff
@@ -120,9 +120,9 @@ By design, d1 is the smallest direction and d2 is the highest
 	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
 		return
 	if(powernet && (powernet.available_power > 0))		// is it powered?
-		to_chat(user, "<span class='danger'>Total power: [DisplayPower(powernet.available_power)]\nLoad: [DisplayPower(powernet.power_demand)]\nExcess power: [DisplayPower(get_surplus())]</span>")
+		to_chat(user, chat_box_examine("<span class='notice'>Total power: [DisplayPower(powernet.available_power)]\nLoad: [DisplayPower(powernet.power_demand)]\nExcess power: [DisplayPower(get_surplus())]</span>"))
 	else
-		to_chat(user, "<span class='danger'>The cable is not powered.</span>")
+		to_chat(user, "<span class='notice'>The cable is not powered.</span>")
```

Comment:
I think they should stay red, warning is probably best
```suggestion
		to_chat(user, "<span class='warning'>The cable is not powered.</span>")
```

Reply:
Oh, that does look even nicer! I was copying how the gas analyzer looks :P